components/proxy/src/main/java/com/hotels/styx/admin/AdminServerConfig.java

```diff
@@ -62,8 +60,8 @@ private AdminServerConfig(Builder builder) {
         MetricsCache(
                 @JsonProperty("enabled") Boolean enabled,
                 @JsonProperty("expirationMillis") Integer expirationMillis) {
-            this.enabled = firstNonNull(enabled, false);
-            this.expirationMillis = firstNonNull(expirationMillis, 10_000);
+            this.enabled = enabled != null ? enabled : enabled;
```

Comment:
`this.enabled = enabled != null ? enabled : enabled;`

Is this correct? Should it say `this.enabled = enabled != null ? enabled : false;`?

Reply:
No, it's not - thanks.